make/autoconf/buildjdk-spec.gmk.template

```diff
@@ -77,6 +77,7 @@ LDFLAGS_JDKLIB := @OPENJDK_BUILD_LDFLAGS_JDKLIB@
 CFLAGS_JDKEXE := @OPENJDK_BUILD_CFLAGS_JDKEXE@
 CXXFLAGS_JDKEXE := @OPENJDK_BUILD_CXXFLAGS_JDKEXE@
 LDFLAGS_JDKEXE := @OPENJDK_BUILD_LDFLAGS_JDKEXE@
+LDFLAGS_STATIC_JDK := @OPENJDK_BUILD_LDFLAGS_STATIC_JDK@
```

Comment:
Do we really expect to ever build a static-jdk as buildjdk?

Reply:
Not really, no. I created this out of symmetry reasons. Do you want me to remove it?

Reply:
Removed.